packages/plugins/ENS/src/SNSAdaptor/SocialAccountList.tsx

```diff
@@ -9,21 +11,21 @@ import useStyles from './useStyles'
 import { useI18N } from '../locales'
 
 interface SocialAccountListProps {
-    restOfValidNextIdTwitterBindings: BindingProof[]
+    validNextIdTwitterBindings: BindingProof[]
 }
 
-export function SocialAccountList({ restOfValidNextIdTwitterBindings }: SocialAccountListProps) {
+export function SocialAccountList({ validNextIdTwitterBindings }: SocialAccountListProps) {
     const t = useI18N()
-    const { classes, cx } = useStyles()
-
-    const [menu, openMenu] = useMenuConfig(
-        restOfValidNextIdTwitterBindings.map((x, i) => (
+    const { classes, cx, theme } = useStyles({ isMenuScroll: validNextIdTwitterBindings.length > 5 })
+    const position = useWindowScroll()
+    const [menu, openMenu, closeMenu] = useMenuConfig(
+        validNextIdTwitterBindings.map((x, i) => (
             <MenuItem
                 className={classes.socialAccountListItem}
                 disabled={false}
                 key={i}
                 onClick={() => openWindow(`https://twitter.com/${x.identity}`)}>
-                <Icons.TwitterRound />
+                <Icons.TwitterRoundWithNoBorder width={20} height={20} />
```

Comment:
TwitterRound has no border either now.

Reply:
![image](https://user-images.githubusercontent.com/52657989/192688423-b7ebd4e5-fa04-4d6f-8f15-712b37dd1a86.png)

Reply:
@zhouhanseng sent this to me since he cannot replace it too. You may adjust it on your side.

Reply:
I think the border should be written in CSS. We have no such an icon in our UIkit.